portality/dao.py

```diff
@@ -96,27 +96,22 @@ def save(self, retries=0, back_off_factor=1, differentiate=False, blocking=False
             app.logger.warn("System is in READ-ONLY mode, save command cannot run")
             return
 
-        if 'id' in self.data:
-            id_ = self.data['id'].strip()
```

Comment:
I've replaced these five lines:
```python
        if 'id' in self.data:
            id_ = self.data['id'].strip()
        else:
            id_ = self.makeid()
            self.data['id'] = id_
```

The variable ```_id``` isn't used anywhere else in the function (according to searches and PyCharm), just to set it in the object. My change just removes the extra fluff.